src/cli.py

```diff
@@ -61,19 +61,22 @@ def add_test_user():
         command += add_secure_config(password_group)
 
     name = get_image_name("montagu-cli", versions.api)
-    args = ["add", "Test User", "test.user", "test.user@imperial.ac.uk", "password", "--if-not-exists"]
-    run(command + [name] + args)
-
-    args = ["addRole", "test.user", "user"]
 
-    run(command + [name] + args)
+    run_cmd(command, name, ["add", "Test User", "test.user", "test.user@imperial.ac.uk", "password", "--if-not-exists"])
+    run_cmd(command, name, ["addRole", "test.user", "user"])
+    run_cmd(command, name, ["addRole", "test.user", "reports-reviewer"])
+    run_cmd(command, name, ["addRole", "test.user", "touchstone-reviewer"])
+    run_cmd(command, name, ["addUserToGroup", "test.user", "ALL"])
```

Comment:
So this user is supposed to be the admin user? Shall we make this clearer and call them "admin.user"? And I guess they don't want to be added to all modelling groups (in fact I think this is what causes them to have an impossibly long token)

Reply:
Renamed to test.admin.